test/manual/mocharc.json

```diff
@@ -1,5 +1,8 @@
 {
-  "require": "ts-node/register",
+  "require": [
+    "ts-node/register",
+    "test/tools/runner/chai_addons.ts"
```

Comment:
Unrelated to the feature but needed for sinon-chai assertions in the manual tests.